samples/grpc/nodejs/package.json

```diff
@@ -11,7 +11,7 @@
     "prettier:fix": "prettier --write \"**/**/!(*.d).{js,json}\""
   },
   "dependencies": {
-    "@eventstore/db-client": "0.0.0-alpha.12",
+    "@eventstore/db-client": "0.0.0-alpha.17",
```

Comment:
Might as well update this to `1.0.0`

Reply:
I missed the release. Bumped in https://github.com/EventStore/documentation/pull/268/commits/95373b1900d7de9fc5732fe7db2cdb4f196584fd.